greenbone/feed/sync/main.py

```diff
@@ -69,13 +70,45 @@ def filter_syncs(
     )
 
 
+def do_selftest(error_console: Console) -> int:
+    """
+    Check for sha256sum and rsync commands.
+    """
+    try:
+        subprocess.run(
+            ["sha256sum", "--help"],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            check=True,
+        )
+    except PermissionError:
+        error_console.print("The sha256sum binary could not be found.")
+        return 1
+
+    try:
+        subprocess.run(
+            ["rsync", "--help"],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            check=True,
+        )
+    except PermissionError:
```

Comment:
Same as above. I think it should be subprocess.CalledProcessError

Reply:
When the file is not found, I get the PermissionError exception.

Reply:
I got a FileNotFoundError. so lets be safe here and use
```
    except (PermissionError, FileNotFoundError, subprocess.SubprocessError):
```